### PR TITLE
Add dicom anonymisation webapp

### PR DESCRIPTION
**Pull request by Centrus007**
#312

### CODE REVIEW

**Comment by SimonBiggs** [comment]
@Centrus007 hope you don't mind, the following should have fixed your issue:

https://github.com/pymedphys/pymedphys/pull/335/commits/806ddc9f7c572aded44bcb89307464f9838c8d87

**Comment by Centrus007** [comment]
@Thank you!

I've been struggling to get any files in output... that issue is still there :(

**Comment by SimonBiggs** [comment]
> @SimonBiggs, thank you! That's one issue sorted!
> 
> I've been struggling to get any files in output... that issue is still there :(

You'll need to rerun `yarn app:wheels`. I need to set up a listener that watches any changes in PyMedPhys library and automatically propagates the code to the app during development.

**Comment by SimonBiggs** [comment]
You may also need to kill and restart the server.

**Comment by SimonBiggs** [comment]
Over at https://deploy-preview-335--app-pymedphys.netlify.com/ I am getting a new error (within the console)

ImportError: cannot import name 'DicomBase' from 'pymedphys.dicom' (/lib/python3.7/pymedphys/dicom.py)

We haven't exposed DicomBase within the API. I'm not sure if we're ready to expose DicomBase through the API also...

Maybe we should be importing just your function? The one that is already on the documentation site.

**Comment by SimonBiggs** [comment]
As a rule, I think any import we use within the app should first be available on the docs (I'm already breaking that rule with trf2csv... ) ... but it would be worthwhile aiming to follow.

I really should get anything I import within the app for the talk into the documentation...

https://pymedphys.com/user/library/dicom.html

**Comment by SimonBiggs** [comment]
Also, just to play around, you can edit the script within the following:

image

(That's a VSCode editor within the browser right there :) )

**Comment by Centrus007** [comment]
DicomBase was the issue. Works now! Will push soon.

This... is.... AMAZING!!!!!!!!!!!!!

**Comment by SimonBiggs** [comment]
> DicomBase was the issue. Works now! Will push soon.
> 
> This... is.... AMAZING!!!!!!!!!!!!!

:D :)

**Comment by SimonBiggs** [comment]
@Centrus007 I'll let you merge once you're ready.

**Comment by Centrus007** [comment]
This'll do for now :)

File: packages/pymedphys/src/pymedphys/dicom.py [hunk 1]
```diff
@@ -13,4 +13,4 @@
     dicom_dataset_from_dict
 )
 
-from pymedphys_analysis.geometry import get_structure_aligned_cube
+# from pymedphys_analysis.geometry import get_structure_aligned_cube
```
Comment (SimonBiggs) [inline packages/pymedphys/src/pymedphys/dicom.py]: @Centrus007 the docs use this don't they?

Comment (SimonBiggs) [reply packages/pymedphys/src/pymedphys/dicom.py]: We just need to remove `get_structure_aligned_cube` from the docs for now I think.

Comment (SimonBiggs) [reply packages/pymedphys/src/pymedphys/dicom.py]: The following really needs to be fixed:

https://github.com/pymedphys/pymedphys/issues/340 ...

**Pull request merged**